sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2639,6 +2638,20 @@ object GpuOverrides {
         override def convertToGpu(): GpuExpression = GpuCollectList(
           childExprs.head.convertToGpu(), c.mutableAggBufferOffset, c.inputAggBufferOffset)
       }),
+    expr[CollectSet](
+      "Collect a set of unique elements, only supported in rolling window in current.",
+      // GpuCollectSet is not yet supported under GroupBy and Reduction context.
+      // Compared to CollectList, StructType is NOT in GpuCollectSet because underlying
```

Comment:
Missing `supported ` ?
```suggestion
      // Compared to CollectList, StructType is NOT supported in GpuCollectSet because underlying
```